### DISCUSSION TITLE
Doris Be node crash ,version:doris-1.2.5

### DISCUSSION THREAD

**Discussion by qiaoshiling** [apache/doris#24015] [A - General / Q&A]
be.out:
*** Query id: 81a8f3828ff845ce-897d25f2e712af6a ***
*** Aborted at 1694055428 (unix time) try "date -d @1694055428" if you are using GNU date ***
*** Current BE git commitID: Unknown ***
*** SIGSEGV address not mapped to object (@0x720000005b) received by PID 14774 (TID 0x7f6caa5e3700) from PID 91; stack trace: ***
 0# doris::signal::(anonymous namespace)::FailureSignalHandler(int, siginfo_t*, void*) at /root/doris/be/src/common/signal_handler.h:420
 1# os::Linux::chained_handler(int, siginfo*, void*) in /usr/local/jdk/jre/lib/amd64/server/libjvm.so
 2# JVM_handle_linux_signal in /usr/local/jdk/jre/lib/amd64/server/libjvm.so
 3# signalHandler(int, siginfo*, void*) in /usr/local/jdk/jre/lib/amd64/server/libjvm.so
 4# 0x00007F6D7C41A400 in /lib64/libc.so.6
 5# _ZNSt8__detail9__variant17__gen_vtable_implINS0_12_Multi_arrayIPFNS0_21__deduce_visit_resultIvEEOZN5doris10vectorized15AggregationNode24_emplace_into_hash_tableEPPcRSt6vectorIPKNS6_7IColumnESaISD_EEmE3$_0RSt7variantIJNS6_27AggregationMethodSerializedI9PHHashMapI9StringRefS8_11DefaultHashISM_vELb0EEEENS6_26AggregationMethodOneNumberIh12FixedHashMapIhS8_28FixedHashMapImplicitZeroCellIhS8_16HashTableNoStateE28FixedHashTableCalculatedSizeISV_E9AllocatorILb1ELb1EEELb0EEENSR_ItSS_ItS8_ST_ItS8_SU_E24FixedHashTableStoredSizeIS12_ESZ_ELb0EEENSR_IjSL_IjS8_9HashCRC32IjELb0EELb0EEENSR_ImSL_ImS8_S17_ImELb0EELb0EEENS6_30AggregationMethodStringNoCacheI13StringHashMapIS8_SZ_EEENSR_INS6_7UInt128ESL_IS1I_S8_S17_IS1I_ELb0EELb0EEENSR_IjSL_IjS8_14HashMixWrapperIjS18_ELb0EELb0EEENSR_ImSL_ImS8_S1M_ImS1B_ELb0EELb0EEENSR_IS1I_SL_IS1I_S8_S1M_IS1I_S1J_ELb0EELb0EEENS6_37AggregationMethodSingleNullableColumnINSR_IhNS6_26AggregationDataWithNullKeyIS10_EELb0EEEEENS1W_INSR_ItNS1X_IS15_EELb0EEEEENS1W_INSR_IjNS1X_IS19_EELb0EEEEENS1W_INSR_ImNS1X_IS1C_EELb0EEEEENS1W_INSR_IjNS1X_IS1O_EELb0EEEEENS1W_INSR_ImNS1X_IS1R_EELb0EEEEENS1W_INSR_IS1I_NS1X_IS1K_EELb0EEEEENS1W_INSR_IS1I_NS1X_IS1U_EELb0EEEEENS1W_INS1E_INS1X_IS1G_EEEEEENS6_26AggregationMethodKeysFixedIS1C_Lb0EEENS2P_IS1C_Lb1EEENS2P_IS1K_Lb0EEENS2P_IS1K_Lb1EEENS2P_ISL_INS6_7UInt256ES8_S17_IS2U_ELb0EELb0EEENS2P_IS2W_Lb1EEENS2P_IS1R_Lb0EEENS2P_IS1R_Lb1EEENS2P_IS1U_Lb0EEENS2P_IS1U_Lb1EEENS2P_ISL_IS2U_S8_S1M_IS2U_S2V_ELb0EELb0EEENS2P_IS34_Lb1EEEEEEJEEESt16integer_sequenceImJLm18EEEE14__visit_invokeESI_S38_ at /var/local/ldb-toolchain/bin/../lib/gcc/x86_64-linux-gnu/11/../../../../include/c++/11/variant:1013
 6# doris::vectorized::AggregationNode::_pre_agg_with_serialized_key(doris::vectorized::Block*, doris::vectorized::Block*) at /root/doris/be/src/vec/exec/vaggregation_node.cpp:1067
 7# std::_Function_handler<doris::Status (doris::vectorized::Block*, doris::vectorized::Block*), std::_Bind_result<doris::Status, doris::Status (doris::vectorized::AggregationNode::*(doris::vectorized::AggregationNode*, std::_Placeholder<1>, std::_Placeholder<2>))(doris::vectorized::Block*, doris::vectorized::Block*)> >::_M_invoke(std::_Any_data const&, doris::vectorized::Block*&&, doris::vectorized::Block*&&) at /var/local/ldb-toolchain/bin/../lib/gcc/x86_64-linux-gnu/11/../../../../include/c++/11/bits/std_function.h:291
 8# doris::vectorized::AggregationNode::get_next(doris::RuntimeState*, doris::vectorized::Block*, bool*) at /root/doris/be/src/vec/exec/vaggregation_node.cpp:506
 9# doris::ExecNode::get_next_after_projects(doris::RuntimeState*, doris::vectorized::Block*, bool*) at /root/doris/be/src/exec/exec_node.cpp:853
10# doris::PlanFragmentExecutor::get_vectorized_internal(doris::vectorized::Block**) at /root/doris/be/src/runtime/plan_fragment_executor.cpp:360
11# doris::PlanFragmentExecutor::open_vectorized_internal() in /opt/soft/doris-1.2.5/be/lib/doris_be
12# doris::PlanFragmentExecutor::open() in /opt/soft/doris-1.2.5/be/lib/doris_be
13# doris::FragmentExecState::execute() at /root/doris/be/src/runtime/fragment_mgr.cpp:261
14# doris::FragmentMgr::_exec_actual(std::shared_ptr<doris::FragmentExecState>, std::function<void (doris::PlanFragmentExecutor*)>) at /root/doris/be/src/runtime/fragment_mgr.cpp:508
15# std::_Function_handler<void (), doris::FragmentMgr::exec_plan_fragment(doris::TExecPlanFragmentParams const&, std::function<void (doris::PlanFragmentExecutor*)>)::$_0>::_M_invoke(std::_Any_data const&) at /var/local/ldb-toolchain/bin/../lib/gcc/x86_64-linux-gnu/11/../../../../include/c++/11/bits/std_function.h:291
16# doris::ThreadPool::dispatch_thread() in /opt/soft/doris-1.2.5/be/lib/doris_be
17# doris::Thread::supervise_thread(void*) at /root/doris/be/src/util/thread.cpp:455
18# start_thread in /lib64/libpthread.so.0
19# clone in /lib64/libc.so.6

Query id: 81a8f3828ff845ce-897d25f2e712af6a

select count(*) from (SELECT DISTINCT * FROM ((SELECT a.sku_code FROM (SELECT adsVevorSkuSiteBasicLabelD.stat_day AS stat_day, adsVevorSkuSiteBasicLabelD.stat_date AS stat_date, adsVevorSkuSiteBasicLabelD.sku_code AS sku_code, adsVevorSkuSiteBasicLabelD.sku_name AS sku_name, adsVevorSkuSiteBasicLabelD.site_code AS site_code, adsVevorSkuSiteBasicLabelD.group_code AS group_code, adsVevorSkuSiteBasicLabelD.group_name AS group_name, adsVevorSkuSiteBasicLabelD.fst_lst_time AS fst_lst_time, adsVevorSkuSiteBasicLabelD.lst_days AS lst_days, adsVevorSkuSiteBasicLabelD.promt_price AS promt_price, adsVevorSkuSiteBasicLabelD.promt_profit_rate AS promt_profit_rate, adsVevorSkuSiteBasicLabelD.feed_title AS feed_title, adsVevorSkuSiteBasicLabelD.lifecycle_name AS lifecycle_name, adsVevorSkuSiteBasicLabelD.is_lst AS is_lst, adsVevorSkuSiteBasicLabelD.is_bulk_prd AS is_bulk_prd, adsVevorSkuSiteBasicLabelD.is_must_buy AS is_must_buy, adsVevorSkuSiteBasicLabelD.revw_cnt AS revw_cnt, adsVevorSkuSiteBasicLabelD.avg_rating AS avg_rating, adsVevorSkuSiteBasicLabelD.wos AS wos, adsVevorSkuSiteBasicLabelD.sales_inv_qty AS sales_inv_qty, adsVevorSkuSiteBasicLabelD.on_trans_mid_inv_qty AS on_trans_mid_inv_qty, adsVevorSkuSiteBasicLabelD.on_trans_last_inv_qty AS on_trans_last_inv_qty, adsVevorSkuSiteBasicLabelD.se_exp_cnt_1d AS se_exp_cnt_1d, adsVevorSkuSiteBasicLabelD.se_exp_cnt_7d AS se_exp_cnt_7d, adsVevorSkuSiteBasicLabelD.se_exp_cnt_14d AS se_exp_cnt_14d, adsVevorSkuSiteBasicLabelD.se_exp_cnt_30d AS se_exp_cnt_30d, adsVevorSkuSiteBasicLabelD.se_exp_cnt_60d AS se_exp_cnt_60d, adsVevorSkuSiteBasicLabelD.se_exp_cnt_180d AS se_exp_cnt_180d, adsVevorSkuSiteBasicLabelD.se_exp_cnt_360d AS se_exp_cnt_360d, adsVevorSkuSiteBasicLabelD.a2w_clk_pv_1d AS a2w_clk_pv_1d, adsVevorSkuSiteBasicLabelD.a2w_clk_pv_7d AS a2w_clk_pv_7d, adsVevorSkuSiteBasicLabelD.a2w_clk_pv_14d AS a2w_clk_pv_14d, adsVevorSkuSiteBasicLabelD.a2w_clk_pv_30d AS a2w_clk_pv_30d, adsVevorSkuSiteBasicLabelD.a2w_clk_pv_60d AS a2w_clk_pv_60d, adsVevorSkuSiteBasicLabelD.a2w_clk_pv_180d AS a2w_clk_pv_180d, adsVevorSkuSiteBasicLabelD.a2w_clk_pv_360d AS a2w_clk_pv_360d, adsVevorSkuSiteBasicLabelD.view_pbd_pv_1d AS view_pbd_pv_1d, adsVevorSkuSiteBasicLabelD.view_pbd_pv_7d AS view_pbd_pv_7d, adsVevorSkuSiteBasicLabelD.view_pbd_pv_14d AS view_pbd_pv_14d, adsVevorSkuSiteBasicLabelD.view_pbd_pv_30d AS view_pbd_pv_30d, adsVevorSkuSiteBasicLabelD.view_pbd_pv_60d AS view_pbd_pv_60d, adsVevorSkuSiteBasicLabelD.view_pbd_pv_180d AS view_pbd_pv_180d, adsVevorSkuSiteBasicLabelD.view_pbd_pv_360d AS view_pbd_pv_360d, adsVevorSkuSiteBasicLabelD.view_pbd_uv_1d AS view_pbd_uv_1d, adsVevorSkuSiteBasicLabelD.view_pbd_uv_7d AS view_pbd_uv_7d, adsVevorSkuSiteBasicLabelD.view_pbd_uv_14d AS view_pbd_uv_14d, adsVevorSkuSiteBasicLabelD.view_pbd_uv_30d AS view_pbd_uv_30d, adsVevorSkuSiteBasicLabelD.view_pbd_uv_60d AS view_pbd_uv_60d, adsVevorSkuSiteBasicLabelD.view_pbd_uv_180d AS view_pbd_uv_180d, adsVevorSkuSiteBasicLabelD.view_pbd_uv_360d AS view_pbd_uv_360d, adsVevorSkuSiteBasicLabelD.a2c_clk_uv_1d AS a2c_clk_uv_1d, adsVevorSkuSiteBasicLabelD.a2c_clk_uv_7d AS a2c_clk_uv_7d, adsVevorSkuSiteBasicLabelD.a2c_clk_uv_14d AS a2c_clk_uv_14d, adsVevorSkuSiteBasicLabelD.a2c_clk_uv_30d AS a2c_clk_uv_30d, adsVevorSkuSiteBasicLabelD.a2c_clk_uv_60d AS a2c_clk_uv_60d, adsVevorSkuSiteBasicLabelD.a2c_clk_uv_180d AS a2c_clk_uv_180d, adsVevorSkuSiteBasicLabelD.a2c_clk_uv_360d AS a2c_clk_uv_360d, adsVevorSkuSiteBasicLabelD.sales_tred_w AS sales_tred_w, adsVevorSkuSiteBasicLabelD.sales_tred_m AS sales_tred_m, adsVevorSkuSiteBasicLabelD.cvr_w_avg AS cvr_w_avg, adsVevorSkuSiteBasicLabelD.dvat_4w AS dvat_4w, adsVevorSkuSiteBasicLabelD.dvat_8w AS dvat_8w, adsVevorSkuSiteBasicLabelD.dvat_12w AS dvat_12w, adsVevorSkuSiteBasicLabelD.is_clr_wms AS is_clr_wms, adsVevorSkuSiteBasicLabelD.dim_cube_code AS dim_cube_code FROM ads_vevor_sku_site_basic_label_d AS adsVevorSkuSiteBasicLabelD) a WHERE (a.site_code = 'UK' AND a.is_bulk_prd > 0)) EXCEPT (SELECT b.sku_code FROM (SELECT adsVevorSkuSitePlmTrdBasicLabelD.stat_day AS stat_day, adsVevorSkuSitePlmTrdBasicLabelD.stat_date AS stat_date, adsVevorSkuSitePlmTrdBasicLabelD.sku_code AS sku_code, adsVevorSkuSitePlmTrdBasicLabelD.sku_name AS sku_name, adsVevorSkuSitePlmTrdBasicLabelD.site_code AS site_code, adsVevorSkuSitePlmTrdBasicLabelD.plm_code AS plm_code, adsVevorSkuSitePlmTrdBasicLabelD.batch_ord_cnt_1d AS batch_ord_cnt_1d, adsVevorSkuSitePlmTrdBasicLabelD.batch_ord_cnt_7d AS batch_ord_cnt_7d, adsVevorSkuSitePlmTrdBasicLabelD.batch_ord_cnt_14d AS batch_ord_cnt_14d, adsVevorSkuSitePlmTrdBasicLabelD.batch_ord_cnt_30d AS batch_ord_cnt_30d, adsVevorSkuSitePlmTrdBasicLabelD.batch_ord_cnt_60d AS batch_ord_cnt_60d, adsVevorSkuSitePlmTrdBasicLabelD.batch_ord_cnt_180d AS batch_ord_cnt_180d, adsVevorSkuSitePlmTrdBasicLabelD.batch_ord_cnt_360d AS batch_ord_cnt_360d, adsVevorSkuSitePlmTrdBasicLabelD.chk_ord_cnt_1d AS chk_ord_cnt_1d, adsVevorSkuSitePlmTrdBasicLabelD.chk_ord_cnt_7d AS chk_ord_cnt_7d, adsVevorSkuSitePlmTrdBasicLabelD.chk_ord_cnt_14d AS chk_ord_cnt_14d, adsVevorSkuSitePlmTrdBasicLabelD.chk_ord_cnt_30d AS chk_ord_cnt_30d, adsVevorSkuSitePlmTrdBasicLabelD.chk_ord_cnt_60d AS chk_ord_cnt_60d, adsVevorSkuSitePlmTrdBasicLabelD.chk_ord_cnt_180d AS chk_ord_cnt_180d, adsVevorSkuSitePlmTrdBasicLabelD.chk_ord_cnt_360d AS chk_ord_cnt_360d, adsVevorSkuSitePlmTrdBasicLabelD.pay_ord_cnt_1d AS pay_ord_cnt_1d, adsVevorSkuSitePlmTrdBasicLabelD.pay_ord_cnt_7d AS pay_ord_cnt_7d, adsVevorSkuSitePlmTrdBasicLabelD.pay_ord_cnt_14d AS pay_ord_cnt_14d, adsVevorSkuSitePlmTrdBasicLabelD.pay_ord_cnt_30d AS pay_ord_cnt_30d, adsVevorSkuSitePlmTrdBasicLabelD.pay_ord_cnt_60d AS pay_ord_cnt_60d, adsVevorSkuSitePlmTrdBasicLabelD.pay_ord_cnt_180d AS pay_ord_cnt_180d, adsVevorSkuSitePlmTrdBasicLabelD.pay_ord_cnt_360d AS pay_ord_cnt_360d, adsVevorSkuSitePlmTrdBasicLabelD.refd_ord_cnt_1d AS refd_ord_cnt_1d, adsVevorSkuSitePlmTrdBasicLabelD.refd_ord_cnt_7d AS refd_ord_cnt_7d, adsVevorSkuSitePlmTrdBasicLabelD.refd_ord_cnt_14d AS refd_ord_cnt_14d, adsVevorSkuSitePlmTrdBasicLabelD.refd_ord_cnt_30d AS refd_ord_cnt_30d, adsVevorSkuSitePlmTrdBasicLabelD.refd_ord_cnt_60d AS refd_ord_cnt_60d, adsVevorSkuSitePlmTrdBasicLabelD.refd_ord_cnt_180d AS refd_ord_cnt_180d, adsVevorSkuSitePlmTrdBasicLabelD.refd_ord_cnt_360d AS refd_ord_cnt_360d, adsVevorSkuSitePlmTrdBasicLabelD.refd_cny_amt_1d AS refd_cny_amt_1d, adsVevorSkuSitePlmTrdBasicLabelD.refd_cny_amt_7d AS refd_cny_amt_7d, adsVevorSkuSitePlmTrdBasicLabelD.refd_cny_amt_14d AS refd_cny_amt_14d, adsVevorSkuSitePlmTrdBasicLabelD.refd_cny_amt_30d AS refd_cny_amt_30d, adsVevorSkuSitePlmTrdBasicLabelD.refd_cny_amt_60d AS refd_cny_amt_60d, adsVevorSkuSitePlmTrdBasicLabelD.refd_cny_amt_180d AS refd_cny_amt_180d, adsVevorSkuSitePlmTrdBasicLabelD.refd_cny_amt_360d AS refd_cny_amt_360d, adsVevorSkuSitePlmTrdBasicLabelD.pay_sku_qty_1d AS pay_sku_qty_1d, adsVevorSkuSitePlmTrdBasicLabelD.pay_sku_qty_7d AS pay_sku_qty_7d, adsVevorSkuSitePlmTrdBasicLabelD.pay_sku_qty_14d AS pay_sku_qty_14d, adsVevorSkuSitePlmTrdBasicLabelD.pay_sku_qty_30d AS pay_sku_qty_30d, adsVevorSkuSitePlmTrdBasicLabelD.pay_sku_qty_60d AS pay_sku_qty_60d, adsVevorSkuSitePlmTrdBasicLabelD.pay_sku_qty_180d AS pay_sku_qty_180d, adsVevorSkuSitePlmTrdBasicLabelD.pay_sku_qty_360d AS pay_sku_qty_360d, adsVevorSkuSitePlmTrdBasicLabelD.pay_cny_amt_1d AS pay_cny_amt_1d, adsVevorSkuSitePlmTrdBasicLabelD.pay_cny_amt_7d AS pay_cny_amt_7d, adsVevorSkuSitePlmTrdBasicLabelD.pay_cny_amt_14d AS pay_cny_amt_14d, adsVevorSkuSitePlmTrdBasicLabelD.pay_cny_amt_30d AS pay_cny_amt_30d, adsVevorSkuSitePlmTrdBasicLabelD.pay_cny_amt_60d AS pay_cny_amt_60d, adsVevorSkuSitePlmTrdBasicLabelD.pay_cny_amt_180d AS pay_cny_amt_180d, adsVevorSkuSitePlmTrdBasicLabelD.pay_cny_amt_360d AS pay_cny_amt_360d, adsVevorSkuSitePlmTrdBasicLabelD.dim_cube_code AS dim_cube_code FROM ads_vevor_sku_site_plm_trd_basic_label_d AS adsVevorSkuSitePlmTrdBasicLabelD) b WHERE (b.plm_code = 'amazon' AND b.refd_cny_amt_7d >= 1 AND b.site_code = 'UK'))) apitempsql ORDER BY sku_code ) ctmp

**Comment by yagagagaga** [comment]
Can you provide the table DDL?

**Comment by qiaoshiling** [comment]
This is my table ddl:
CREATE TABLE `ads_vevor_sku_site_basic_label_d` (
  `stat_day` varchar(200) NOT NULL COMMENT '',
  `stat_date` datetime NULL COMMENT '',
  `sku_code` varchar(200) NOT NULL COMMENT '',
  `sku_name` varchar(200) NULL COMMENT '',
  `site_code` varchar(200) NOT NULL COMMENT '',
  `group_name` varchar(200) NULL COMMENT '',
  `fst_lst_time` datetime NULL COMMENT '',
  `lst_days` bigint(20) NULL COMMENT '',
  `promt_price` decimal(20, 2) NULL COMMENT '',
  `promt_profit_rate` decimal(20, 2) NULL COMMENT '',
  `feed_title` varchar(1000) NULL COMMENT '',
  `lifecycle_name` varchar(200) NULL COMMENT '',
  `is_lst` varchar(200) NULL COMMENT '',
  `is_bulk_prd` varchar(200) NULL COMMENT '',
  `is_must_buy` varchar(200) NULL COMMENT '',
  `revw_cnt` bigint(20) NULL COMMENT '',
  `avg_rating` decimal(20, 2) NULL COMMENT '',
  `wos` decimal(20, 2) NULL COMMENT '',
  `sales_inv_qty` bigint(20) NULL COMMENT '',
  `on_trans_inv_qty` bigint(20) NULL COMMENT '',
  `se_exp_cnt_1d` bigint(20) NULL COMMENT '',
  `se_exp_cnt_7d` bigint(20) NULL COMMENT '',
  `se_exp_cnt_14d` bigint(20) NULL COMMENT '',
  `se_exp_cnt_30d` bigint(20) NULL COMMENT '',
  `se_exp_cnt_60d` bigint(20) NULL COMMENT '',
  `se_exp_cnt_180d` bigint(20) NULL COMMENT '',
  `se_exp_cnt_360d` bigint(20) NULL COMMENT '',
  `a2w_clk_pv_1d` bigint(20) NULL COMMENT '',
  `a2w_clk_pv_7d` bigint(20) NULL COMMENT '',
  `a2w_clk_pv_14d` bigint(20) NULL COMMENT '',
  `a2w_clk_pv_30d` bigint(20) NULL COMMENT '',
  `a2w_clk_pv_60d` bigint(20) NULL COMMENT '',
  `a2w_clk_pv_180d` bigint(20) NULL COMMENT '',
  `a2w_clk_pv_360d` bigint(20) NULL COMMENT '',
  `view_pbd_pv_1d` bigint(20) NULL COMMENT '',
  `view_pbd_pv_7d` bigint(20) NULL COMMENT '',
  `view_pbd_pv_14d` bigint(20) NULL COMMENT '',
  `view_pbd_pv_30d` bigint(20) NULL COMMENT '',
  `view_pbd_pv_60d` bigint(20) NULL COMMENT '',
  `view_pbd_pv_180d` bigint(20) NULL COMMENT '',
  `view_pbd_pv_360d` bigint(20) NULL COMMENT '',
  `view_pbd_uv_1d` bigint(20) NULL COMMENT '',
  `view_pbd_uv_7d` bigint(20) NULL COMMENT '',
  `view_pbd_uv_14d` bigint(20) NULL COMMENT '',
  `view_pbd_uv_30d` bigint(20) NULL COMMENT '',
  `view_pbd_uv_60d` bigint(20) NULL COMMENT '',
  `view_pbd_uv_180d` bigint(20) NULL COMMENT '',
  `view_pbd_uv_360d` bigint(20) NULL COMMENT '',
  `a2c_clk_uv_1d` bigint(20) NULL COMMENT '',
  `a2c_clk_uv_7d` bigint(20) NULL COMMENT '',
  `a2c_clk_uv_14d` bigint(20) NULL COMMENT '',
  `a2c_clk_uv_30d` bigint(20) NULL COMMENT '',
  `a2c_clk_uv_60d` bigint(20) NULL COMMENT '',
  `a2c_clk_uv_180d` bigint(20) NULL COMMENT '',
  `a2c_clk_uv_360d` bigint(20) NULL COMMENT '',
  `sales_tred_w` varchar(1000) NULL COMMENT '',
  `sales_tred_m` varchar(1000) NULL COMMENT '',
  `cvr_w_avg` decimal(20, 2) NULL COMMENT '',
  `dvat_4w` decimal(20, 2) NULL COMMENT '',
  `dvat_8w` decimal(20, 2) NULL COMMENT '',
  `dvat_12w` decimal(20, 2) NULL COMMENT '',
  `is_clr_wms` varchar(200) NULL COMMENT '',
  `dim_cube_code` varchar(200) NULL COMMENT ''
) ENGINE = OLAP DUPLICATE KEY(`stat_day`) COMMENT '-' DISTRIBUTED BY HASH(`stat_day`, `sku_code`, `site_code`) BUCKETS 16 PROPERTIES (
  "replication_allocation" = "tag.location.default: 1",
  "in_memory" = "false",
  "storage_format" = "V2",
  "enable_unique_key_merge_on_write" = "true",
  "disable_auto_compaction" = "false"
);

**Reply by yagagagaga** [reply]
This DDL does not match the SQL.

**Reply by qiaoshiling** [reply]
sorry , I forget other one table ddl :
CREATE TABLE `ads_vevor_sku_site_plm_trd_basic_label_d` (
  `stat_day` varchar(200) NULL COMMENT '',
  `stat_date` datetime NULL COMMENT '',
  `sku_code` varchar(200) NULL COMMENT '',
  `sku_name` varchar(200) NULL COMMENT '',
  `site_code` varchar(200) NULL COMMENT '',
  `plm_code` varchar(200) NULL COMMENT '',
  `batch_ord_cnt_1d` bigint(20) NULL COMMENT '',
  `batch_ord_cnt_7d` bigint(20) NULL COMMENT '',
  `batch_ord_cnt_14d` bigint(20) NULL COMMENT '',
  `batch_ord_cnt_30d` bigint(20) NULL COMMENT '',
  `batch_ord_cnt_60d` bigint(20) NULL COMMENT '',
  `batch_ord_cnt_180d` bigint(20) NULL COMMENT '',
  `batch_ord_cnt_360d` bigint(20) NULL COMMENT '',
  `chk_ord_cnt_1d` bigint(20) NULL COMMENT '',
  `chk_ord_cnt_7d` bigint(20) NULL COMMENT '',
  `chk_ord_cnt_14d` bigint(20) NULL COMMENT '',
  `chk_ord_cnt_30d` bigint(20) NULL COMMENT '',
  `chk_ord_cnt_60d` bigint(20) NULL COMMENT '',
  `chk_ord_cnt_180d` bigint(20) NULL COMMENT '',
  `chk_ord_cnt_360d` bigint(20) NULL COMMENT '',
  `pay_ord_cnt_1d` bigint(20) NULL COMMENT '',
  `pay_ord_cnt_7d` bigint(20) NULL COMMENT '',
  `pay_ord_cnt_14d` bigint(20) NULL COMMENT '',
  `pay_ord_cnt_30d` bigint(20) NULL COMMENT '',
  `pay_ord_cnt_60d` bigint(20) NULL COMMENT '',
  `pay_ord_cnt_180d` bigint(20) NULL COMMENT '',
  `pay_ord_cnt_360d` bigint(20) NULL COMMENT '',
  `refd_ord_cnt_1d` bigint(20) NULL COMMENT '',
  `refd_ord_cnt_7d` bigint(20) NULL COMMENT '',
  `refd_ord_cnt_14d` bigint(20) NULL COMMENT '',
  `refd_ord_cnt_30d` bigint(20) NULL COMMENT '',
  `refd_ord_cnt_60d` bigint(20) NULL COMMENT '',
  `refd_ord_cnt_180d` bigint(20) NULL COMMENT '',
  `refd_ord_cnt_360d` bigint(20) NULL COMMENT '',
  `refd_cny_amt_1d` decimal(20, 2) NULL COMMENT '',
  `refd_cny_amt_7d` decimal(20, 2) NULL COMMENT '',
  `refd_cny_amt_14d` decimal(20, 2) NULL COMMENT '',
  `refd_cny_amt_30d` decimal(20, 2) NULL COMMENT '',
  `refd_cny_amt_60d` decimal(20, 2) NULL COMMENT '',
  `refd_cny_amt_180d` decimal(20, 2) NULL COMMENT '',
  `refd_cny_amt_360d` decimal(20, 2) NULL COMMENT '',
  `pay_sku_qty_1d` bigint(20) NULL COMMENT '',
  `pay_sku_qty_7d` bigint(20) NULL COMMENT '',
  `pay_sku_qty_14d` bigint(20) NULL COMMENT '',
  `pay_sku_qty_30d` bigint(20) NULL COMMENT '',
  `pay_sku_qty_60d` bigint(20) NULL COMMENT '',
  `pay_sku_qty_180d` bigint(20) NULL COMMENT '',
  `pay_sku_qty_360d` bigint(20) NULL COMMENT '',
  `pay_cny_amt_1d` decimal(20, 2) NULL COMMENT '',
  `pay_cny_amt_7d` decimal(20, 2) NULL COMMENT '',
  `pay_cny_amt_14d` decimal(20, 2) NULL COMMENT '',
  `pay_cny_amt_30d` decimal(20, 2) NULL COMMENT '',
  `pay_cny_amt_60d` decimal(20, 2) NULL COMMENT '',
  `pay_cny_amt_180d` decimal(20, 2) NULL COMMENT '',
  `pay_cny_amt_360d` decimal(20, 2) NULL COMMENT '',
  `dim_cube_code` varchar(200) NULL COMMENT ''
) ENGINE = OLAP DUPLICATE KEY(`stat_day`) COMMENT '' DISTRIBUTED BY HASH(`stat_day`, `sku_code`) BUCKETS 16 PROPERTIES (
  "replication_allocation" = "tag.location.default: 1",
  "in_memory" = "false",
  "storage_format" = "V2",
  "enable_unique_key_merge_on_write" = "true",
  "disable_auto_compaction" = "false"
);

**Reply by yagagagaga** [reply]
This is indeed a bug, you can upgrade to 2.0.1 to solve this problem.
